python/lsst/ip/diffim/makeKernel.py

```diff
@@ -330,7 +332,7 @@ def makeKernelBasisList(self, targetFwhmPix=None, referenceFwhmPix=None,
 
         return basisList
 
-    def _buildCellSet(self, templateMaskedImage, scienceMaskedImage, candidateList):
+    def _buildCellSet(self, templateMaskedImage, scienceMaskedImage, candidateList, imageBBox):
```

Comment:
This seems like an unnecessary interface change to me:
1)  You don't use your `bbox` variable again in code that calls it after line 123. 
2) `imageBBox` can be exactly determined from the other two inputs already there:  `templateMaskedImage` and `scienceMaskedImage`.

Recommendation would be, right before constructing the SpatialCellSet in 
`kernelCellSet = lsst.afw.math.SpatialCellSet(imageBBox, sizeCellX, sizeCellY)` below

you could:
```
        imageBBox = templateMaskedImage.getBBox()
        imageBBox.clip(scienceMaskedImage.getBBox()
``` 
and thats it.